Add `Capture The Achievement 3` News Post (#14784)
ppy/osu-wiki: news/2026/2026-06-07-capture-the-achievement-3.md

---
layout: post
title: Capture The Achievement 3
date: 2026-06-07 20:00:00 +0000
series: online_events
---

Capture The Achievement returns this Summer with 75 new achievements to solve from July 3rd to July 13th! 

![](/wiki/shared/news/2026-06-07-capture-the-achievement-3/banner.jpg)

> If this is your first time hearing of Capture The Achievement (CTA), it's a team event (1-5 players per team) where teams complete "achievements" to earn points, and the team with the most points wins. Each achievement has its own unique solution that requires some mixture of knowledge, research, technical and mechanical skill, and sometimes sheer dumb luck. The event is much more fun with a team, so I recommend gathering some friends or looking at the "what if I don't have a team" section.

Reviewer: "I" is used almost never

```suggestion
If this is your first time hearing of Capture The Achievement (CTA), it's a team event (1–5 players per team) where teams complete "achievements" to earn points, and the team with the most points wins. Each achievement has its own unique solution that requires some mixture of knowledge, research, technical and mechanical skill, and sometimes sheer dumb luck. The event is much more fun with a team, so gather some friends or take a look at the ["what if I don't have a team" section](#what-if-i-don't-have-a-team?).
```

The last iteration of CTA took place last summer with 100 achievements across 16 days. This year, we've shortened the length and decreased the number of achievements to reduce burnout for the competitive teams and focus on quality, respectively. In addition, we tweaked the scoring, tag system, and achievement release schedule based on feedback.

> The event will take place on the [CTA website](https://cta.sheppsu.me/) and you can join the [Discord server](https://discord.gg/kuE5mTZgqK) to see announcements, ask any specific questions, or interact with fellow players. 

Reviewer: nit but it can technically be faster to click a direct link

```suggestion
The event will take place on the [CTA website](https://cta.sheppsu.me/) and you can join the [Discord server](https://discord.com/invite/kuE5mTZgqK) to see announcements, ask any specific questions, or interact with fellow players. 
```

## Schedule & Achievement Release

| Phase | Day |
| --: | :-- |

> | Registration closes | June 15th (0:00 UTC) |

Reviewer: we typically format it like this to remove any possible confusion

```suggestion
| Registration closes | June 14th (23:59 UTC) |
```

| Event starts | July 3rd (18:00 UTC) |
| Event ends | July 13th (18:00 UTC) |

In total there will be 75 achievements, but they will release in "batches" throughout the event. Each batch consists of 5 achievements of varying difficulty. The first batch will release when the event starts, and each subsequent batch will release every 16 hours. 

> Between the event starting and ending, you are free to complete achievements whenever and spend however much time you wish. Beware though, if you play to win, prepare yourself to compete with some real sweats.

Reviewer: ```suggestion
Between the event starting and ending, you are free to complete achievements whenever and spend however much time you wish. Beware though: if you play to win, prepare yourself to compete with some real sweats!
```

## Prizes

| Placement | Prizes |
| :-: | :-- |
| 1st | Profile badge (pending) and 4 months of osu!supporter for each team member |
| 2nd | 2 months of osu!supporter for each team member |
| 3rd | 1 month of osu!supporter for each team member |

## What game mode is this event for?

> All game modes! Most achievements will be completable using any game mode, however, some achievements will require playing a certain game mode.

Reviewer: ```suggestion
All game modes! Most achievements will be completable using any game mode. However, some achievements will require playing a certain game mode.
```

## How do I participate?

> Go to the [CTA website](https://cta.sheppsu.me/) and login by clicking the osu! logo in the top right (or in the header dropdown on mobile). On your dashboard page, there will be a button to register. Assuming you pass screening, you'll be eligible to participate. From there, you can create and join teams or change your free agent settings.

Reviewer: ```suggestion
Go to the [CTA website](https://cta.sheppsu.me/) and log in by clicking the osu! logo in the top right (or in the header dropdown on mobile). On your dashboard page, there will be a button to register. Assuming you pass screening, you'll be eligible to participate. From there, you can create and join teams or change your free agent settings.
```

## What if I don't have a team?

> There are a few options. Firstly, there is a free agent system to match you with a random team. After registering, you may opt into being a free agent and specify whether you plan on playing casually or competitively; this will help match you to a team. You can also try joining the [Discord server](https://discord.gg/kuE5mTZgqK) and using the #looking-for-team channel. Lastly, you could play solo, but I don't recommend it because it's less fun.

Reviewer: a dash or something else is more appropriate than a semicolon here

```suggestion
There are a few options. Firstly, there is a free agent system to match you with a random team. After registering, you may opt into being a free agent and specify whether you plan on playing casually or competitively — this will help match you to a team. You can also try joining the [Discord server](https://discord.com/invite/kuE5mTZgqK) and using the `#looking-for-team` channel. As a last resort, you could try playing on your own, but that's less fun.
```

## What does an achievement look like?

An achievement has a title, description, tags, and possibly 1 or more beatmaps attached. You'll also see the number of completions and how many points it's currently worth.

> The first image shows what was a "chat" achievement in CTA2, but we'll be phasing these out in favor of "password" achievements. The second image shows a password achievement, which has a text input to guess the password. A correct password input will give you the completion. You may also notice the difference in border colors. The red border indicates your team has not completed the achievement. The green border indicates that your team has completed it, which is also why the submit button is greyed out.

Reviewer: :gb:

```suggestion
The first image shows what was a "chat" achievement in CTA2, but we'll be phasing these out in favour of "password" achievements. The second image shows a password achievement, which has a text input field to guess the password. A correct password input will give you the completion. You may also notice the difference in border colors. The red border indicates that your team has not completed the achievement, while the green border indicates that your team has already completed it.
```

> ![Achievement Example 1](/wiki/shared/news/2026-06-07-capture-the-achievement-3/achievement1.jpg)

Reviewer: these are technically treated as being on the same line but wrapped to two lines due to the image width. the backslash inserts a `<br>` tag

```suggestion
![Achievement Example 1](/wiki/shared/news/2026-06-07-capture-the-achievement-3/achievement1.jpg)\
```

![Achievement Example 2](/wiki/shared/news/2026-06-07-capture-the-achievement-3/achievement2.jpg)

If you're curious about the solution to **The Everlasting Epigrapher**, you can [click here](https://cta.sheppsu.me/achievements/314).

If you'd like to check out more previous achievements, you can go to the list of [CTA2 achievements](https://cta.sheppsu.me/iterations/2/achievements) and [CTA1 achievements](https://cta.sheppsu.me/iterations/1/achievements).

---

> If you have any unanswered questions, try checking the FAQ section on the [CTA website](https://cta.sheppsu.me/) or joining the [Discord server](https://discord.gg/kuE5mTZgqK) and asking there (feel free to ping/direct message me @sheppsu).

Reviewer: ```suggestion
If you have any unanswered questions, try checking the FAQ section on the [CTA website](https://cta.sheppsu.me/) or joining the [Discord server](https://discord.com/invite/kuE5mTZgqK) and asking there (feel free to ping/dm me @sheppsu).
```
